frontend/app/components/api/api-v3/hal-resources/work-package-resource.service.ts

```diff
@@ -268,12 +268,22 @@ export class WorkPackageResource extends HalResource {
     this.updateForm(this.$source)
       .then(form => {
         var payload = this.mergeWithForm(form);
+        const sentValues = Object.keys(this.$pristine);
```

Comment:
I agree that this is a workaround. A sound solution requires reworking the way `$pristine` works, and possibly to queue / block subsequent requests while one is running.